Source/Core/InputCommon/ControllerEmu.h

```diff
@@ -50,27 +51,27 @@ class ControllerEmu
     class Control
     {
     protected:
-      Control(ControllerInterface::ControlReference* const _ref, const std::string& _name)
+      Control(ControlReference* const _ref, const std::string& _name)
```

Comment:
This is another section that I haven't read through at all, and has a ton of these sort of violations. Is it alright to leave this alone for now?